ADI_model/Pipfile

```diff
@@ -8,6 +8,7 @@ numpy = "*"
 matplotlib = "*"
 imageio = "*"
 scipy = "*"
+scikit-rf = ">=0.24"
```

Comment:
As already mentioned, version 0.24 is not released yet. I excpect a release in the next weeks.

Reply:
The requested version is released now:
https://github.com/scikit-rf/scikit-rf/releases/tag/v0.24.0